codespell_lib/data/dictionary.txt

```diff
@@ -1303,16 +1303,35 @@ alignrigh->alignright
 alikes->alike, likes,
 aline->align, a line, line, saline,
 alined->aligned
+aling->align, along, a line, ailing, sling,
+alinged->aligned
+alinging->aligning
 alingment->alignment
+alings->aligns, slings,
 alinment->alignment
 alinments->alignments
 alising->aliasing
 aliver->alive, liver, a liver, sliver,
 allcate->allocate
+allcateing->allocating
+allcater->allocator
+allcaters->allocators
+allcating->allocating
 allcation->allocation
 allcator->allocator
+allcator->allocator
```

Comment:
CI is complaining that `allcator` appears twice :wink:

Reply:
oops.....thank you

Reply:
https://github.com/codespell-project/codespell/pull/1815/commits/4677b714757cf9d98dbbdef6a3d2f358634f9a3c